dev-python/sphinx-autoapi/sphinx-autoapi-1.8.4.ebuild

```
@@ -0,0 +1,32 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7..10} )
+
+inherit distutils-r1
+
+DESCRIPTION="A new approach to API documentation in Sphinx."
+HOMEPAGE="https://sphinx-autoapi.readthedocs.io/"
+SRC_URI="https://github.com/readthedocs/sphinx-autoapi/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+# SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz" # doesn't include docs/
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+python" # go javascript dotnet, currently not supported
+
+RDEPEND="
+	dev-python/sphinx[${PYTHON_USEDEP}]
+	python? ( dev-python/astroid[${PYTHON_USEDEP}] )
+"
+
+DOCS=( README.rst CHANGELOG.rst )
+HTML_DOCS=( html/. )
+
+distutils_enable_tests pytest
+
+src_compile() {
+	sphinx-build -c docs/ -b html docs/ html/
```

[You must || die on external commands or you won't know if they failed](https://devmanual.gentoo.org/ebuild-writing/error-handling/#the-die-function):
```suggestion
	sphinx-build -c docs/ -b html docs/ html/ || die
```